ghost/core/core/server/data/migrations/versions/5.11/2022-08-23-13-41-backfill-members-created-events.js

```diff
@@ -0,0 +1,30 @@
+const ObjectID = require('bson-objectid').default;
+const logging = require('@tryghost/logging');
+
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const members = await knex('members')
+            .select('id', 'created_at');
+
+        const toInsert = [];
+
+        // eslint-disable-next-line no-restricted-syntax
+        for (const member of members) {
+            toInsert.push({
+                id: ObjectID().toHexString(),
+                member_id: member.id,
+                created_at: member.created_at,
+                source: 'member'
+            });
+        }
+
+        logging.info(`Inserting ${toInsert.length} members created events`);
+        await knex.batchInsert('members_created_events', toInsert);
+    },
+    async function down(knex) {
+        logging.info(`Clearing all members created events`);
+        await knex('members_created_events').truncate();
```

Comment:
We usually use `.del()` for this kind of operation - though `.truncate()` is more correct - do you know if there's a perf improvement with this? NABD either way IMO

Reply:
Truncate should be a little bit faster, don't think we'll notice :)

Reply:
But truncate doesn't support rollbacks, we might need that?

Reply:
If that's the case, let's stick with `del()` for now :)